packages/flutter/lib/src/material/tabs.dart

```diff
@@ -954,6 +959,7 @@ class _TabBarState extends State<TabBar> {
     // If indicatorSize is TabIndicatorSize.label, _tabKeys[i] is used to find
     // the width of tab widget i. See _IndicatorPainter.indicatorRect().
     _tabKeys = widget.tabs.map((Widget tab) => GlobalKey()).toList();
+    _labelPaddings = widget.tabs.map<EdgeInsetsGeometry>((Widget tab) => EdgeInsets.zero).toList();
```

Comment:
I believe, this and the previous line could be simplified if List.filled is used: https://master-api.flutter.dev/flutter/dart-core/List/List.filled.html

Reply:
should i also change the code about _tabKeys ?

Reply:
we can't do the same thing with _tabKeys,  List.filled method fill with same GlobalKey, but widget require unique key.  i have done changes with _labelPaddings.